File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityForStatementTest.cpp
#include "ActivityForStatementTest.h"
#include <alf/library/BasicInputOutput.hpp>
#include <string>

void ForStatementTest()
{
  // print 5,6,7,8,9
  for (int i = 5; i <= 9; i++)
  {
    alf::library::BasicInputOutput::WriteLine(std::to_string(i));
  }
}

File: solutions/alfi.compitest/source_gen/alfi/compitest/ActivityForStatementTest.cs
/// <filename>
///     ActivityForStatementTest.cs
/// </filename>
using Alf.Library;
using System;

public class ActivityForStatementTest
{
    public static void Execute()
    {
        // print 5,6,7,8,9
        for (int i = 5; i <= 9; i++)
        {
            BasicInputOutput.WriteLine(Convert.ToString(i));
        }
    }

    public static void Main(string[] args)
    {
        ActivityForStatementTest.Execute();
    }
}
